plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -801,7 +818,7 @@ def create_vm(module, proxmox, vmid, newid, node, name, memory, cpu, cores, sock
             return False
     elif module.params['clone'] is not None:
         for param in valid_clone_params:
-            if module.params[param] is not None:
+            if module.params[param] is not None and module.params[param] != 'unspecified':
```

Comment:
How about not doing this change (this will remove any usage of `unspecified` in any of the `valid_clone_params`, which is potentially very wrong), but instead modifying `module.params['format']` after `AnsibleModule` is created by replacing `unspecified` with `None` there?

Reply:
Good point. Thank you.